ormconfig.json

```diff
@@ -2,9 +2,9 @@
    "type": "postgres",
    "host": "localhost",
    "port": 5432,
-   "username": "angeloantolini",
-   "password": "Skimboard159753",
-   "database": "First-docker",
+   "username": "local",
+   "password": "postgres",
+   "database": "local",
```

Comment:
Usando esse arquivo fica um pouco mais difícil de chavear entre conectar com o banco local e o de testes do jeito que fizemos nas reuniões (passando os parâmetros no `createConnection`). Mas se vc conseguir uma solução pra isso, sem problemas.

Reply:
Eu fiz primeiramente desse jeito que era mais fácil para focar nos problemas dessa issue em específico e quando for preciso alternar os bancos vou implementar um jeito parecido com o da reunião